contributor_roadmap.md

```diff
@@ -0,0 +1,180 @@
+# Contributor Roadmap/Expectations
+Hello, thank you for your interest in contributing to Matplotlib!
+
+Matplotlib is primarily administered via github, so more information
+on the exact permissions granted to each role is available at [github
+help](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level)
+
+
+
+
+# Development
+Responsible for the codebase (including documentation)
+
+* https://github.com/orgs/matplotlib/teams/developers
+
+
+**contribute**: https://matplotlib.org/devdocs/devel/index.html
+
+**triage:**
+- triaging issues and pull requests means:
+    - assigning labels, milestones, and reviewers
+    - closing and reopening as needed
+    - marking duplicates
+- get triage privileges: granted on 1st merged pull request
+    - (We need to write the script for this, run as a service on heroku’s free tier?  can we do this with actions?)
```

Comment:
It might be possible with actions - let me dig into this and if so I can get working on this ✨

Reply:
Thanks!!!

Reply:
That would be great @trallard!